cpp/README.md

```diff
@@ -88,21 +88,27 @@ Debug build with unit tests:
     $ cd build-debug
     $ cmake -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTS=ON ..
     $ make -j8       # if you have 8 CPU cores, otherwise adjust, use -j`nproc` for all cores
-    $ make test      # to run the tests
+```
+
+After building, you can run the unit tests with:
+
+```bash
+    $ git clone https://github.com/apache/incubator-graphar-testing.git testing  # download the testing data
+    $ GAR_TEST_DATA=${PWD}/testing ctest
```

Comment:
I would recommend to create a folder `dev/` and provide bash/python scripts for downloading the data and set up all the variables. We can do it in a separate PR. It is done like this in [spark](https://github.com/apache/spark/tree/master/dev) and I found it very user-friendly.

Reply:
The suggestion is great, I can do it in the PR #507 .